packages/core/src/order/order.ts

```diff
@@ -84,6 +84,12 @@ export interface OrderConsignment {
     shipping: OrderShippingConsignment[];
 }
 
+export interface OrderShippingConsignmentDiscount {
+    id: number;
+    amount: number;
+    code: string | null;
```

Comment:
Is it null or undefined?

Reply:
The GET order API is returning null or string.

https://github.com/bigcommerce/bigcommerce/blob/f7b7ee97eca5d23d2408767bf22434413926e0d3/app/Http/Storefront/Api/Order/Consignment/Shipping/AppliedShippingDiscountResponseTransformer.php#L24

![image](https://github.com/user-attachments/assets/1e29fe8f-9b3c-4cc3-99bd-60fe51c0af57)